mw-webapp/src/utils/auth/writeNewUserData.ts

```diff
@@ -0,0 +1,17 @@
+import {ref, set} from "firebase/database";
+import {db} from "src/firebase";
+
+interface UserData {
+  userId: string
+  userEmail: string | null
+  userName: string | null
+}
+
+export const writeNewUserData = ({userId, userEmail, userName}: UserData) => {
```

Comment:
Is it possible don't use destructure? If image that user has 100 properties it will be hard to write it all

Reply:
Done